cms/src/components/images/ImageEditorThumbnail.vue

```diff
@@ -8,6 +8,7 @@ import fallbackImage from "@/assets/fallback-image-cms.webp";
 type Props = {
     imageFileCollection?: ImageFileCollectionDto;
     imageUploadData?: ImageUploadDto;
+    editable?: boolean;
```

Comment:
Please rename to `disabled` to keep the naming convention.
(The name `editable` is used elsewhere in the code to name DTO objects that can be edited)

Reply:
Thanks.